Makefile.am

```diff
@@ -43,9 +43,7 @@ AM_CFLAGS = @STRICT_CFLAGS@
 # Make sure to run Gtk-doc tests and build the documentation when doing
 # 'make distcheck'
 AM_DISTCHECK_CONFIGURE_FLAGS = \
-	--enable-gtk-doc \
-	--enable-gir-doc \
-	--enable-strict-flags \
+	--enable-gtk-doc
```

Comment:
Don't do `VARIABLE = $(NULL)`, do `VARIABLE =` instead. The `$(NULL)` is only there at the end of multiline lists so that you don't accidentally put a backslash after your last item and combine it with whatever unrelated code comes after your list.